packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -504,12 +520,16 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
    * @dev The v,r and s signature should be a signed message by the beneficiary
    *      encrypting the authorized address.
    */
-  function authorizeValidatorSigner(address signer, uint8 v, bytes32 r, bytes32 s)
+  function authorizeValidatorSigner(address payable signer, uint8 v, bytes32 r, bytes32 s)
     external
     nonReentrant
     onlyCanValidate
     onlyWhenInProperState
   {
+    // If no previous signer has been authorized, fund the new signer so that tx fees can be paid.
+    if (getAccounts().getValidatorSigner(address(this)) == address(this)) {
+      fundSigner(signer);
+    }
```

Comment:
is this something we really only want to ReleaseGold?  And do other contracts even use these account contract methods?

Otherwise, I don't see the advantage of having this logic here here where you need to make two requests to the accounts contract rather than letting the accounts contract handle this.

Reply:
Yeah I think only for release gold. Once we hit mainnet, folks can just fund their signing keys themselves, but while transfers are disabled that won't be possible